Packages/com.unity.inputsystem/Documentation~/Migration.md

```diff
@@ -34,202 +44,108 @@ There are scripting symbols defined which allow you to use conditional compilati
 All of the new APIs listed below are in the `UnityEngine.InputSystem` namespace. The namespace is omitted here for brevity. `UnityEngine.InputSystem` is referenced in full for easy disambiguation.
```

Comment:
This was already present but slightly confused why it says 

The namespace is omitted here for brevity

but then says 
`UnityEngine.InputSystem` is referenced in full for easy disambiguation.

I feel we could drop the last line
`UnityEngine.InputSystem` is referenced in full for easy disambiguation.

Reply:
yes - I'm not sure where that last line came from actually, maybe left over from a previous version of the page.